Gems/ScriptAutomation/Code/Source/Utils/Utils.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/std/string/string.h>
+
+namespace ScriptAutomation
+{
+    namespace Utils
+    {
+        bool SupportsResizeClientArea();
+        void ResizeClientArea(uint32_t width, uint32_t height);
+
+        bool SupportsToggleFullScreenOfDefaultWindow();
+        void ToggleFullScreenOfDefaultWindow();
+
+        //! retrieve the default script screenshots output folder
+        AZStd::string GetScreenshotsPath(bool resolvePath);
+
+        //! retrieve the default script profiling data output folder
+        AZStd::string GetProfilingPath(bool resolvePath);
+
+        //! Provides a more convenient way to call AZ::IO::FileIOBase::GetInstance()->ResolvePath()
+        AZStd::string ResolvePath(const AZStd::string& path);
```

Comment:
Also I believe a Path could be converted to an AZStd::string by using a functor support for the BehaviorContext `Method` function as seen with this [example](https://github.com/o3de/o3de/blob/development/Code/Editor/PythonEditorFuncs.cpp#L1119)